packages/tupaia-web-server/src/app/createApp.ts

```diff
@@ -39,6 +41,7 @@ export function createApp() {
     .get<ProjectRequest>('project/:projectCode', handleWith(ProjectRoute))
     .get<UserRequest>('getUser', handleWith(UserRoute))
     .get<DashboardsRequest>('dashboards', handleWith(DashboardsRoute))
+    .get<EntitiesRequest>('entities/:hierarchyName/:rootEntityCode', handleWith(EntitiesRoute))
```

Comment:
I figure since `hierarchyName` is required we should have it as a url param rather than a query option